automation_tools/__init__.py

```diff
@@ -958,17 +958,19 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     execute(setenforce, selinux_mode, host=host)
     execute(install_tasks[distribution], host=host)
 
-    if distribution in ('cdn', 'downstream', 'iso'):
+    if distribution in (
+            'satellite6-cdn', 'satellite6-downstream', 'satellite6-iso'):
         execute(setup_default_capsule, host=host)
 
-    execute(setup_default_docker, host=host)
-    # execute returns a dict, the result is the first value. OS version is the
-    # resulting index 1
-    if execute(distro_info, host=host).values()[0][1] == 6:
-        with warn_only():
+    if not distribution.startswith('sam'):
```

Comment:
Would it be more straightforward to get rid of the negative and write this instead?

    if distribution.startswith('satellite6'):